lib/pages/image_selector/image_selector.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+
+class ImageSelector extends StatefulWidget {
+  const ImageSelector({Key? key}) : super(key: key);
+
+  @override
+  State<ImageSelector> createState() => _ImageSelectorState();
+}
+
+class _ImageSelectorState extends State<ImageSelector> {
+  File? imageFile;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Capturing Images'),
+        centerTitle: true,
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(12.0),
+        child: imageCap(),
+      ),
+    );
+  }
+
+  Column imageCap() {
+    return Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          if (imageFile != null)
+            Container(
+              width: 640,
+              height: 480,
```

Comment:
Magic numbers. These could be specified as a variable. 

Example: `Size containerSize = const Size(640, 480);`
`width: containerSize.width,
height: containerSize.height,`